cl/users/models.py

```diff
@@ -177,31 +177,31 @@ class Meta:
         verbose_name_plural = "user profiles"
 
 
-class SUB_TYPES(object):
-    """SNS Event Subtypes"""
+class EMAIL_NOTIFICATIONS(object):
+    """SES Email Notifications Subtypes"""
 
     UNDETERMINED = 0
     GENERAL = 1
-    NOEMAIL = 2
+    NO_EMAIL = 2
     SUPPRESSED = 3
-    ONACCOUNTSUPPRESSIONLIST = 4
-    MAILBOXFULL = 5
-    MESSAGETOOLARGE = 6
-    CONTENTREJECTED = 7
-    ATTACHMENTREJECTED = 8
+    ON_ACCOUNT_SUPPRESSIONLIST = 4
```

Comment:
I think `ON_ACCOUNT_SUPPRESSIONLIST` should be `ON_ACCOUNT_SUPPRESSION_LIST`

Reply:
Changed